agreements/ddl.sql

```diff
@@ -0,0 +1,41 @@
+/*
+Title       Database creation script for Agreements database design
+Date        May 26th 2020
+Author      Trevor Cummings
+Description This file is a script to create the tables for the Agreements database. The data model for Agreements will be 
+            deployed in a postgreSQL database.
+            
+            
+*/
+
+CREATE TABLE commercial_agreements (

```

Comment:
Few missing fields / related tables perhaps according to latest [API defintion](https://raw.githubusercontent.com/Crown-Commercial-Service/ccs-scale-api-definitions/master/agreements/agreements-service.yaml):

- Agreement -> `benefits` (list of strings)
- Lot -> `description` (string)
- Lot -> `type` (enum `product|service|product and service`)
- Lot -> `sectors` (list of strings)
- Lot -> `routesToMarket`, `buyerNeeds` and `rules` (complex types, see API spec for detail)

Easiest way to visualise the API spec is to copy the entire thing into the online [Swagger Editor](https://editor.swagger.io/) and navigate through the Schemas section on the right hand side.

Reply:
I hasten to add - we don't need all this yet.. 👍

Reply:
Are they missing in the agreements ddl? The agreements DDL is just holding what is needed for FaT and not the agreements service. The data model for the Agreement service is here https://app.lucidchart.com/documents/edit/ffb89d3c-ee65-4ab2-b5cb-e42ca8213204/0_0

Reply:
I think we need to agree on one communication channel as Google Hangouts, Slack, email and GitHub is sending my head into a spin.

Reply:
Github primarily for review comments and discussion on code (with further expansion on Slack as required) 👍

Reply:
> I think we need to agree on one communication channel as Google Hangouts, Slack, email and GitHub is sending my head into a spin.

I can set up a WhatsApp group to discuss further?

Reply:
To clarify requirements for FaT Phase 1, we need the following additional columns at Lot level I believe (from the API):

- `type` (enum `bat`, `cat`)
- `routeToMarket` (enum `fc`, `da`)
- `scale` (boolean)